### PR TITLE
ci: fix classic ci

### PR DESCRIPTION
**Pull request by ntindle**
### Background

We moved from master -> dev for the mainline development branch without making sure it triggers the ci. This updates all the instances of development to dev for branch names. **It does not attempt to verify the CI functions as needed under the new scheme**

### Changes 🏗️

- find replace "development" -> "dev"
- formating due to saving file in vscode
- Don't error if folder exists in classic dockerfiles

Only for the new autogpt platform, currently in autogpt_platform/

- Create from scratch and execute an agent with at least 3 blocks
- Import an agent from file upload, and confirm it executes correctly
- Upload agent to marketplace
- Import an agent from marketplace and confirm it executes correctly
- Edit an agent from monitor, and confirm it executes correctly

### CODE REVIEW

**Review by Pwuts — COMMENTED** [review]
A million unrelated changes, pls no

File: .github/workflows/classic-autogpt-docker-ci.yml [hunk 1]
```diff
@@ -2,17 +2,17 @@ name: Classic - AutoGPT Docker CI
 
 on:
   push:
-    branches: [ master, development ]
+    branches: [master, dev]
     paths:
-      - '.github/workflows/classic-autogpt-docker-ci.yml'
```
Comment (Swiftyos) [inline .github/workflows/classic-autogpt-docker-ci.yml:7]: Do you know how come your formatter replaces ' with " ?
It make the diff harder to follow - though if we can all get our formatters to work the same, its find to keep this change in the PR

File: .github/workflows/classic-autogpt-docker-ci.yml [hunk 2]
```diff
@@ -2,17 +2,17 @@ name: Classic - AutoGPT Docker CI
...
-      - 'classic/original_autogpt/**'
-      - 'classic/forge/**'
+      - ".github/workflows/classic-autogpt-docker-ci.yml"
+      - "classic/original_autogpt/**"
+      - "classic/forge/**"
   pull_request:
-    branches: [ master, dev, release-* ]
+    branches: [ master, dev, release-*
```
Comment (Swiftyos) [inline .github/workflows/classic-autogpt-docker-ci.yml]: it looks like the trailing bracket is missing

**Comment by Pwuts** [comment]
@aarushik93 this PR fixes a couple of references to `develop` that you missed in that PR

**Comment by ntindle** [comment]
> I didn't realise you had a PR for this already... but I merged one yesterday - does that fix what you are also trying to fix or completely different?
> 
> [a2ef456](https://github.com/Significant-Gravitas/AutoGPT/commit/a2ef456525c0d1a979fd2ecf7e8951ac785d48b6)

Imo your pr was great because it got the ones we use often working immediately

**Comment by dagelf** [comment]
If it's broken, what does it matter? ...... at least people following the instructions linked in the original issue won't be able to accidentally wipe their environments.

**Comment by aarushik93** [comment]
@ntindle what's the status on this one?

File: .github/workflows/classic-autogpt-docker-cache-clean.yml [hunk 3]
```diff
@@ -15,46 +15,46 @@ jobs:
...
-        new_commits_json: ${{ null }}
-        compare_url_template: ${{ format('/{0}/compare/{{base}}...{{head}}', github.repository) }}
-
-        github_context_json: ${{ toJSON(github) }}
-        job_env_json: ${{ toJSON(env) }}
-        vars_json: ${{ toJSON(vars) }}
-
-      run: .github/workflows/scripts/docker-ci-summary.sh >> $GITHUB_STEP_SUMMARY
-      continue-on-error: true
+      - name: Checkout repository
```
Comment (majdyz) [inline .github/workflows/classic-autogpt-docker-cache-clean.yml:18]: I assume the tabbing was expected?

**Review by majdyz — APPROVED** [review]

File: .github/workflows/classic-autogpt-docker-cache-clean.yml (hunk 3 above)
Comment (ntindle) [reply .github/workflows/classic-autogpt-docker-cache-clean.yml:18]: Yes

**Pull request merged**